### DISCUSSION TITLE
How do I delete the example components and stories that come with a fresh Storybook install?

### DISCUSSION THREAD

**Discussion by AdamDSherman** [storybookjs/storybook#23736] [General]
I'm new to Storybook (and React in general). I've just installed Storybook 7.2.1 into my Create React App project and have fired it up. It works well, however I am having 2 issues:

1. I cannot create Components or Stories with the same name (but different directories) as the example stories.
2. I cannot figure out how to delete the example stories.

If I delete any of the example story or component files Storybook will just crash. If I clear Node cache and rebuild Storybook I get the following error in my Terminal:

```
resolve './Button' in '/Users/me/Desktop/React/react-test/src/stories'
  using description file: /Users/me/Desktop/React/react-test/package.json (relative path: ./src/stories)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Users/me/Desktop/React/react-test/package.json (relative path: ./src/stories/Button)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/me/Desktop/React/react-test/src/stories/Button doesn't exist
      .web.mjs
```

Would anyone know what I need to do to proceed?

**Comment by domyen** [comment]
That's strange that it crashes, most folks delete the directory with the examples.

**Comment by chantastic** [comment]
is there a chance you can post the code online?

i can speak to the "stories by the same name in different directories" thing tho…

if you use the `title: 'MyComponent'` property in component meta, that is what sets the story path.
that has to be unique.
so you you need to put those titles in "categories" like `"GroupA/MyComponent"` and `"GorupB/MyComponent"`.

if you want to defer to file-system-based "categories", you can delete the `title` prop in favor of `component`.

just remember that one of those has to be present — either `title` or `component`.

i prefer `component`. but sometimes i'll use `title`, if i'm creating a story file that generates stories for multiple components.
